lib/engine/step/g_18_sj/special_buy.rb

```diff
@@ -27,7 +27,7 @@ def process_buy_special(action)
 
         def setup
           super
-          @items << Item.new(description: 'GKB bonus', cost: 0)
+          @gkb_bonus_item = Item.new(description: 'GKB bonus', cost: 0)
```

Comment:
In 1856 ||= is used instead of =. Probably does not matter.

Reply:
1856 used `||=` because setup gets called several times during the game so keeping one consistent `Item` is useful, I'm changing this to use `||=` too